### ISSUE TITLE
How to converts a sumo flow file, eg. cross.trips.xml, to CityFlow format?

### ISSUE THREAD

**Issue by chuckbassx** [cityflow-project/CityFlow#91]
I have read your docs, in the 'Roadnet File Format' chapter,  it mentioned 'python converter.py' can convert a sumo roadnet file to CityFlow format. 
And do you have some code to convert a <sumo flow file> to CityFlow format?

**Comment by only-changer** [comment]
Yes, we do have a converter. see it in https://github.com/cityflow-project/CityFlow/tree/master/tools/converter

**Comment by chuckbassx** [comment]
Sorry , I mean convert a 'SUMO FLOW FILE', eg. cross.trips.xml, to CityFlow format (such as cross_trip.json), NOT convert 'sumo roadnet file'

**Comment by mainakchain** [comment]
Can we make the cityflow format flow file in anyway? I don't find it anywhere in the documentation or any example. I do have the roadnet file but I am not able to get the flow file. Please help @chuckbassx @only-changer.

**Comment by sinaKGT** [comment]
i have same problem but didn't find any solution to that, unfortunately this repo is only tool for their author to create paper and didn't update in last 5 years and answer the users' problems.
